modules/nf-core/checkv/downloaddatabase/tests/main.nf.test

```diff
@@ -0,0 +1,24 @@
+nextflow_process {
+
+    name "Test Process CHECKV_DOWNLOADDATABASE"
+    script "../main.nf"
+    process "CHECKV_DOWNLOADDATABASE"
+    options "-stub"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "checkv"
+    tag "checkv/downloaddatabase"
+
+    test("No input: download only") {
+
+        then {
+            assertAll(
+                { assert process.success },
```

Comment:
might as well check for the presence of all the files.

Reply:
@Joon-Klaps I ran into linting errors related to the md5 of empty files (since this is a stub run). Is there a way to bypass those?

Reply:
Soon™ https://github.com/nf-core/tools/pull/2571